velero-2-mixin/README.md

```diff
@@ -17,7 +17,7 @@ and the following alerts:
 
 ## Velero Cluster Overview
 
-The Velero cluster overview dashboard a top level overview that provides details on alerts, volume snapshots, backup and restore counts.
+The Velero cluster view dashboard provides details on alerts, volume snapshots, backup, and restore counts.
 
 ![Velero Overview Dashboard]()
```

Comment:
Line 22 and 28 have the same dashboard name, one should be for the cluster, one should be for the overview.